chapter03/c03_flan.rpy

```diff
@@ -74,35 +74,35 @@ label c03_flan:
-                t "Hm..."
-                t "I don't like the sound of this guy."
-                c "Hm?"
-                Toma's eyes shifted to the side as we walked, a slight look of disgust in his eyes.
-                t "I mean, if he's staring at you then I can give a guess as to why, which kind of bothers me."
-                t "Especially since you're in that... [Pause] Everyone knows that there's nothing underneath."
-                Toma's face reddened a little bit as he began to furiously scratch his head, his teeth gritting in distress.
-                t "Ugh, when is that stupid thing going to come in?!"
-                c "Um, Toma? [Pause] Are you alright?"
-                t "Huh?"
-                t "Oh, I'm fine."
-                t "But hey, do me a favor and don't get too close to him, got it?"
-                t "If he keeps on staring at you, let me know and I'll tell the guy to stop."
-                c "Huh? [Pause] Um, well alright."
-                I smiled in confusion at Toma as we continued down the hall, soon talking about what drink to get.
+            c "The new resident is a young boy about my age and... [Pause] He's very...strange."
+            t "Huh? [Pause] Strange how and in what way?"
+            c "Well, he is really quiet and mostly stared at me when I talked with him."
+            t "He stared at you?"
+            c "Basically, yes. [Pause] His eyes were screaming distrust and wariness towards me."
+            c "I think that being in the accident has really hurt his ability to trust people."
+            c "But because of that... [Pause] I really want to help him." 
+            c "He seems to be only about my age after all, so he shouldn't really have to go through this...being in a hospital."
+            t "Hm..."
+            t "I don't like the sound of this guy."
+            c "Hm?"
+            Toma's eyes shifted to the side as we walked, a slight look of disgust in his eyes.
+            t "I mean, if he's staring at you then I can give a guess as to why, which kind of bothers me."
+            t "Especially since you're in that... [Pause] Everyone knows that there's nothing underneath."
+            Toma's face reddened a little bit as he began to furiously scratch his head, his teeth gritting in distress.
+            t "Ugh, when is that stupid thing going to come in?!"
+            c "Um, Toma? [Pause] Are you alright?"
+            t "Huh?"
+            t "Oh, I'm fine."
+            t "But hey, do me a favor and don't get too close to him, got it?"
+            t "If he keeps on staring at you, let me know and I'll tell the guy to stop."
+            c "Huh? [Pause] Um, well alright."
+            I smiled in confusion at Toma as we continued down the hall, soon talking about what drink to get.
 
     [Transition]
-    It was now six eighteen in the evening, the hospital hallway lights dimmed down to settle a time passing atmosphere.
```

Comment:
Not sure if you were after figurative language here but I didn't quite understand